src/style.css

```diff
@@ -1,3 +1,112 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
 }
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+/* Menu 1200px */
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.header {
+  margin: 0;
+  height: 60px;
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  margin: 0 auto 0 0;
+  margin-right: auto;
```

Comment:
The `margin: 0 auto 0 0;` and `margin-right: auto;` are redundant. You can simplify this by using just `margin-right: auto;` to achieve the same effect.